include/deal.II/grid/grid_out.h

```diff
@@ -1081,22 +1081,23 @@ class GridOut
 
   /**
    * Write triangulation in VTU format for each processor, then combine these
-   * files into a .pvtu file. The input <tt>view_levels</tt> is set to false by
-   * default, allowing only the active level cells to be written into the .vtu
-   * file, but can be set as true to view each level of a multilevel method.
-   * The input <tt>include_artificial</tt> is also set to false by default,
-   * allowing the processor to only write in owned and ghost cells into its .vtu
-   * file, but can be set as true to view the artificial cells for each processor.
-   * Each .vtu and .pvtu file will have the attributes subdomain, lvlsubdomain, level,
-   * and proc_writing. The level value can be used to seperate the image into the
-   * view of the gird on each level of a multilevel method and the proc_writing value
-   * can be used to seperate the image into each processors owned and ghost cells.
+   * files into a .pvtu file. The output is in the form 
+   * <tt>filename_without_extension.proc000*.vtu</tt> where * is 0,1,...,n_proc and 
+   * <tt>filename_without_extension.pvtu</tt>. The input <tt>view_levels</tt> can be 
+   * set as true to view each level of a multilevel method. The input 
+   * <tt>include_artificial</tt> can be set as true to view the artificial cells for 
+   * each processor. Each .vtu and .pvtu file will have the attributes subdomain, 
+   * lvlsubdomain, level, and proc_writing. The level value can be used to seperate the 
+   * image into the view of the grid on each level of a multilevel method and the 
+   * proc_writing value can be used to seperate the image into each processors owned and 
+   * ghost cells. This is accomplished by applying the "warp by scalar" filter in paraview 
+   * to each of the values.
    */
```

Comment:
Do we want to include an image? Here are two simple ones I used to test and make sure it was working properly.
![levels](https://cloud.githubusercontent.com/assets/11428698/14476617/ce3e36ee-00d6-11e6-9cb0-ee24588ef6cf.png)
![active](https://cloud.githubusercontent.com/assets/11428698/14476616/ce3d6700-00d6-11e6-8119-9e051d9017f2.png)